### PR TITLE
[Keyboard] keyboards/anavi: Add ANAVI Macro Pad 2

### PR DESCRIPTION
**Pull request by leon-anavi**
ANAVI Macro Pad 2 is an open source mini mechanical keyboard with
2 keys and backlit. Powered by ATtiny 85 microcontroller and with
microUSB connector. Designed with KiCad.

Provides the following keymaps for ANAVI Macro Pad 2:

- Default (with a, b and combo to control the LEDs)
- Copy & paste
- Tap dance example with a, b and c
- Mute
- Volume
- Zoom
- Jitsi Meet

Signed-off-by: Leon Anavi <leon@anavi.org>

## Description

Add new keyboard ANAVI Macro Pad 2 and several keymaps for it.

ANAVI Macro Pad 2 is an open source mini mechanical keyboard with 2 keys and 2 LEDs for backlit. Powered by ATtiny 85 microcontroller and with microUSB connector. The PCB has been designed with the free and open source software KiCad.

This patch provides the following keymaps for ANAVI Macro Pad 2:

- Default (with a, b and combo to control the LEDs)
- Copy & paste
- Tap dance example with a, b and c
- Mute
- Volume
- Zoom
- Jitsi Meet

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/anavi/macropad2/keymaps/copypaste/keymap.c [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/anavi/macropad2/keymaps/copypaste/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/anavi/macropad2/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/anavi/macropad2/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/anavi/macropad2/keymaps/jitsi/keymap.c [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/anavi/macropad2/keymaps/jitsi/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/anavi/macropad2/keymaps/mute/keymap.c [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/anavi/macropad2/keymaps/mute/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/anavi/macropad2/keymaps/tap_dance/config.h [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#pragma once
```
Comment (drashna) [inline keyboards/anavi/macropad2/keymaps/tap_dance/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/anavi/macropad2/keymaps/tap_dance/keymap.c [hunk 6]
```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/anavi/macropad2/keymaps/tap_dance/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/anavi/macropad2/keymaps/zoom/keymap.c [hunk 7]
```diff
@@ -0,0 +1,21 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/anavi/macropad2/keymaps/zoom/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by leon-anavi** [comment]
Hi @drashna,

Thank you for the feedback. Following your recommendations I have added GPL2+ compatible license header to all .c files for the keymaps.
 
 Best regards, Leon

File: keyboards/anavi/macropad2/rules.mk [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (drashna) [inline keyboards/anavi/macropad2/rules.mk:28]:
```suggestion
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
UNICODE_ENABLE = no         # Unicode
BLUETOOTH_ENABLE = no       # Enable Bluetooth 
AUDIO_ENABLE = no           # Audio output
```

**Comment by leon-anavi** [comment]
@drashna thank you. I've fix it, added your credentials as `Co-authored-by` git trailer and forced push the new version of the patch.

Thanks, Leon

**Review by drashna — APPROVED** [review]

**Review by spidey3 — APPROVED** [review]

**Pull request merged**